### PR TITLE
adding network to dao connect

### PR DESCRIPTION
**Pull request by brossetti1**
## GitHub Issue

fixes https://github.com/HausDAO/daohaus-monorepo/issues/535

## Changes

- [X] adding `daoChainId` and `setDaoChainId` to the daohaus connect feature
- [X] setting the `daoChainId` on connect in the `core-app`
- [X] adjusting network display and unknown network display

## Packages Added

N/A

## Checks

Before making your PR, please check the following:

- [x] Critical lint errors are resolved
- [x] App runs locally
- [x] App builds locally (run the build command for *any impacted package* and check for any errors before the PR)

# Details

Screen Shot 2022-08-25 at 10 11 01 AM
Screen Shot 2022-08-25 at 10 11 14 AM
Screen Shot 2022-08-25 at 10 11 25 AM

### CODE REVIEW

File: libs/daohaus-connect-feature/src/HausConnectContext.tsx [hunk 1]
```diff
@@ -47,7 +48,8 @@ export type UserConnectType = {
   networks: NetworkConfigs;
   switchNetwork: (chainId: string) => void;
   isProfileLoading: boolean;
-  isDaoScope: boolean;
+  setDaoChainId: React.Dispatch<SetStateAction<string | null>>;
```
Comment (jordanlesich) [inline libs/daohaus-connect-feature/src/HausConnectContext.tsx:51]: We've got a ReactSetter<> utility type to make this cleaner. Don't have to use it here, but for future reference.

File: libs/daohaus-connect-feature/src/components/ConnectButton/UserConnetedDropdown.tsx [hunk 2]
```diff
@@ -54,9 +56,7 @@ export const UserConnectedDropdown = ({ isSm }: { isSm: boolean }) => {
                 </ParMd>
               )}
               <ParXs color={theme.button.primary.text}>
-                {chainId && validNetwork
-                  ? `@${getNetworkName(chainId)}`
-                  : 'Wrong Network'}
+                {(networkName && `@${networkName}`) || 'Unknown'}
```
Comment (jordanlesich) [inline libs/daohaus-connect-feature/src/components/ConnectButton/UserConnetedDropdown.tsx:59]: Wouldn't 'Wrong Network' communicate the point more effectively to the user than 'Unknown"?

File: libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx [hunk 3]
```diff
@@ -54,13 +51,12 @@ export const getNetworkPanels = (
     );
   });
 
-export const NotDaoNetwork = ({ isSm }: { isSm: boolean }) => {
+export const NotchainId = ({ isSm }: { isSm: boolean }) => {
   //  In the future, this will come from the dao context
```
Comment (jordanlesich) [inline libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx:55]: We can probably ditch this comment now

File: libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx [hunk 4]
```diff
@@ -54,13 +51,12 @@ export const getNetworkPanels = (
     );
   });
 
-export const NotDaoNetwork = ({ isSm }: { isSm: boolean }) => {
+export const NotchainId = ({ isSm }: { isSm: boolean }) => {
```
Comment (jordanlesich) [inline libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx:54]: I prefer NotDaoNetwork, tbh.

**Review by jordanlesich — APPROVED** [review]

File: libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx (hunk 4 above)
Comment (brossetti1) [reply libs/daohaus-connect-feature/src/components/NetworkButton/NetworkButton.tsx:54]: whoops that was a find and replace i forgot to change back

**Pull request merged**